CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## v6.0.2
+* [Set configFilePath when reading config file](https://github.com/TypeStrong/ts-loader/pull/942) (#939) @konpikwastaken
```

Comment:
Minor nit - could this be ` - thanks  @konpikwastaken` please? It's ts-loader policy to thank all contributors publicly 🥰

Reply:
Done.